### DISCUSSION TITLE
imports

### DISCUSSION THREAD

**Discussion by gin-gitaxias** [corazawaf/coraza#757] [Q&A] [answered]
Hey, i am surrely missing something but none of my imports works from github :
github.com/corazawaf/coraza/v3
github.com/corazawaf/coraza/v2
github.com/corazawaf/coraza/v3/seclang

i am just following doc but nothing work

**Comment by gin-gitaxias** [comment]
![ok](https://user-images.githubusercontent.com/104769769/229831072-874fa047-d806-4127-8594-ecb117c11fae.png)

**Comment by jcchavezs** [comment]
Are you using the latest version of the library? Feels like you are using
an old version which indeed calls the seclang package that is internal now.

**Reply by gin-gitaxias** [reply]
I am using the doc in coraza.io.

oh and btw i am a bit new to github so i don't know if you saw but i added another comment in my pull request earlier there is still some "coraza-waf" in coraza.io

**Comment by jptosso** [comment]
We have to update coraza.io documentation

**Comment by gin-gitaxias** [comment]
package main
import (
  "github.com/corazawaf/coraza/v3"
  "github.com/corazawaf/coraza/v3/seclang"
)
func parseRules(waf *coraza.Waf){
  parser := seclang.NewParser(waf)
  if err := parser.FromString(`SecAction "id:1,phase:1,deny:403,log"`); err != nil {
    panic(err)
  }
}


i've used this, it come from the doc

**Comment by gin-gitaxias** [comment, marked as answer]
Nevemind my low skill in go is the problem i am really sorry. thank for your help
